.github/workflows/ci_action.yml

```diff
@@ -108,7 +108,11 @@ jobs:
           sentinelhub.config \
             --sh_client_id "${{ secrets.SH_CLIENT_ID }}" \
             --sh_client_secret "${{ secrets.SH_CLIENT_SECRET }}"
-          pytest -m "not geopedia" --cov --cov-report=term --cov-report=xml
+          if [ ${{ github.event_name }} == 'push' ]; then
```

Comment:
in what situations are now tests faster? most of the tests were push-related anyway, were they not?

Reply:
Most tests are pull-request based, and those are the ones that we need ASAP. push triggers only on develop and master (so one per MR)